application/src/main/java/org/cardanofoundation/ledgersync/service/impl/genesis/GenesisDataServiceImpl.java

```diff
@@ -88,6 +88,40 @@ public class GenesisDataServiceImpl implements GenesisDataService {
     public static final String UPDATE_QUORUM = "updateQuorum";
 
     public static final String DELEGATION_KEYS = "genDelegs";
+    private final static String POOL_VOTING_THRESHOLDS = "poolVotingThresholds";
+    private final static String COMMITTEE_NORMAL = "committeeNormal";
+    private final static String COMMITTEE_NO_CONFIDENCE = "committeeNoConfidence";
+    private final static String HARD_FORK_INITIATION = "hardForkInitiation";
+    private final static String MOTION_NO_CONFIDENCE = "motionNoConfidence";
+    private final static String PP_SECURITY_GROUP = "ppSecurityGroup";
+    private final static String D_REP_VOTING_THRESHOLDS = "dRepVotingThresholds";
+    private final static String MOTION_NO_CONFIDENCE1 = "motionNoConfidence";
+    private final static String COMMITTEE_NORMAL1 = "committeeNormal";
```

Comment:
Should we use some prefix like DVT_COMMITEE_NORMAL instead of COMMITTEE_NORMAL1 ?

Same for COMMITTEE_NO_CONFIDENCE1